app/views/aspect_memberships/_aspect_membership_dropdown.mobile.haml

```diff
@@ -0,0 +1,13 @@
+- if user_signed_in? && @person != current_user.person
+  %select{:id => "user_aspects", 'data-person-id' => @person.id}
```

Comment:
Mmh, I wonder if this has to be an ID or if we can't have this reusable already for when somebody wants to add more than one dropdown to a single page, think contacts management/overview.